grafana-plugin/src/containers/MobileAppConnection/parts/DownloadIcons/index.tsx

```diff
@@ -17,7 +17,7 @@ const DownloadIcons: FC = () => (
     <Text type="primary" strong>
       Download
     </Text>
-    <Text type="primary">The Grafana IRM app is available on both the App Store and Google Play Store.</Text>
+    <Text type="primary">The Grafana OnCall app is available on both the App Store and Google Play Store.</Text>
```

Comment:
general question. I see the app is published as "Grafana OnCall" in the app store. If Grafana Incident is considering adding features to the app in the future, what impact does this have on the naming of the app?

Reply:
I think once we integrate Grafana Incindent we'll probably rename the app

Reply:
that's a questions to answer during the next few months and will certainly involve our product and marketing team.  for now, it's the OnCall app.

Reply:
that makes sense, thanks for the clarification guys 👍